spec/controllers/api_controller_spec.rb

```diff
@@ -17,4 +17,44 @@
     expect(response_json).to include("docx")
     expect(response_json).to include("json")
   end
+
+  describe "export" do
+    it 'exports single json' do
+      create(:organization, name: "organization 1")
+      create(:organization, name: "organization 42", id: 42)
+      get :export_organization, id: 42, format: :json
+      expect(response_json.size()).to eq(1)
+      expect(response_json[0]['name']).to include("organization 42")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.